lib/src/wrappers/ModalWrapper.jsx

```diff
@@ -62,7 +63,7 @@ export default class ModalWrapper extends PureComponent {
     onOpen: undefined,
     onClose: undefined,
     onSetToday: undefined,
-    container: undefined,
+    DialogProps: undefined,
```

Comment:
Don't forget to mention this breaking change in release notes ;)

Reply:
+

Reply:
@cherniavskii yep of cource. Thank you)